docs/mcf_format.md

```diff
@@ -316,7 +318,34 @@ typeOf: SurveyItem
 dcid: C:SomeDataset->SurveyItem_Dcid
 ```
 
-Where E is short for "Entity" and "C" is short for "Column".
+Where "E" is short for "Entity" and "C" is short for "Column".
+
+"E" should be used to refrence another entity that is defined in the template
+MCF only. Instead "C" should be used to refer to property values that are 
+entities already populated in the graph and not being defined in the template 
```

Comment:
nit: minor reword suggestion... feel free to ignore :)

Instead "C" should be used to refer to property values represented in columns of the `.csv` file. The values may include:
* References to entities that are already in the graph, and not defined in the Template MCF
* Constants (strings, numbers)
* Enumerations defined in the schema
* Quantities, like [10 20 Years]